### ISSUE TITLE
[local_auth] update to allow intl 0.20.0

### ISSUE THREAD

**Issue by TheSin-** [flutter/flutter#159591]
### What package does this bug report belong to?

local_auth

### What target platforms are you seeing this bug on?

iOS

### Have you already upgraded your packages?

Yes

### Dependency versions

<details><summary>pubspec.lock</summary>

```lock
local_auth: ^2.3.0
```

</details>

### Steps to reproduce

I picked iOS cause I have no choice but this will affect all projects.

I'd like to be able to update to intl 0.20.0

```
Because no versions of local_auth match >2.3.0 <3.0.0 and local_auth 2.3.0 depends on local_auth_darwin ^1.4.0, local_auth ^2.3.0 requires local_auth_darwin ^1.4.0.
And because local_auth_darwin >=1.2.2 depends on intl >=0.17.0 <0.20.0, local_auth ^2.3.0 requires intl >=0.17.0 <0.20.0.
So, because ia_insider depends on both intl ^0.20.0 and local_auth ^2.3.0, version solving failed.
```

### Expected results

allow it

### Actual results

doesn't allow it

### Code sample

<details open><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Videos

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[Paste your output here]
```

</details>

**Comment by mohsinnaqvi606** [comment]
The new version is released [0.20.1](https://pub.dev/packages/intl/changelog) please make it compatible with the latest version

**Comment by TheSin-** [comment]
I agree ;)
